templates/asset.phtml

```diff
@@ -27,7 +27,11 @@
             ][$data['support_level']]) ?>"><?php echo esc(ucfirst($data['support_level'])) ?></span>
         </h4>
 
-        <p class="text-muted">Submitted by user <?php echo esc($data['author']) ?>; <?php echo esc($data['cost']) ?></p>
+        <p class="text-muted">
+          Submitted by user <a href="../asset?filter=<?php echo esc($data['author']) ?>" title="Search assets with filter '<?php echo esc($data['author']) ?>'"><?php echo esc($data['author']) ?></a>;
```

Comment:
Use the "user" parameter instead of the "filter" parameter.

Not sure about this one, but maybe we should also make it add to the current filter?